packages/@o3r/apis-manager/schematics/ng-add/index.ts

```diff
@@ -0,0 +1,42 @@
+import { chain, noop, Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import { NgAddSchematicsSchema } from './schema';
+
+import * as path from 'node:path';
+
+/**
+ * Add Otter apis manager to an Angular Project
+ *
+ * @param options
+ */
+export function ngAdd(options: NgAddSchematicsSchema): Rule {
+  return async (tree: Tree, context: SchematicContext) => {
+    try {
+      const { ngAddPackages, getPeerDepWithPattern, applyEsLintFix, getProjectFromTree } = await import('@o3r/schematics');
+      const { updateApiDependencies } = await import('../helpers/update-api-deps');
+      const deps = getPeerDepWithPattern(path.resolve(__dirname, '..', '..', 'package.json'), /^@(o3r|ama-sdk)/)
+        .filter(peerDep => peerDep !== '@o3r/core'
+          && peerDep !== '@o3r/schematics'
+          && peerDep !== '@o3r/dev-tools');
+      const rulesToExecute = [];
+      const workspaceProject = getProjectFromTree(tree, options.projectName || undefined);
+      if (workspaceProject.projectType === 'application') {
+        rulesToExecute.push(updateApiDependencies());
+      }
+
+      return chain([
+        ...rulesToExecute,
+        options.skipLinter ? noop : applyEsLintFix(),
+        ngAddPackages(deps, { skipConfirmation: true, parentPackageInfo: '@o3r/apis-manager' })
```

Comment:
The `parentPackageInfo: '@o3r/apis-manager'` parameter should be deduced from the the `package.json` to avoid duplication of the package name